core-bom/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'io.micronaut.build.internal.bom'
+}
+
+group projectGroupId
+version projectVersion
+
+repositories {
+    mavenCentral()
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
can we move this to dependencyManagement block?

Reply:
Actually I think that's redundant now